src/PhaseState.jl

```diff
@@ -181,12 +182,12 @@ export getKcs
 Calculates the forward and reverse rate coefficients for a given reaction, phase and state
 Maintains diffusion limitations if the phase has diffusionlimited=true
 """
-@inline function getkfkrev(rxn::ElementaryReaction,ph::U,T::W1,P::W2,C::W3,N::W4,ns::Q1,Gs::Q2,diffs::Q3,V::W5,phi::W8;kf::W6=-1.0,f::W7=-1.0) where {U<:AbstractPhase,W8,W6,W7,W5,W4,W1,W2,W3<:Real,Q1,Q2,Q3<:AbstractArray}
+@inline function getkfkrev(rxn::ElementaryReaction,ph::U,T::W1,P::W2,C::W3,N::W4,ns::Vector{Q1},Gs::Q2,diffs,V::W5,phi::W8;kf::W6=-1.0,f=-1.0) where {U<:AbstractPhase,W8,W6,W5,W4,W1,W2,W3,Q1<:Real,Q2}
     if signbit(kf) 
-        if signbit(f)
-            kf = getkf(rxn,ph,T,P,C,ns,V,phi)
-        else
+        if isa(f,Num) || !signbit(f)
```

Comment:
Can you run a formatting script over your changes or this file?

Reply:
I'm having difficulty getting the formatter extension to connect with the julia kernel. Was there anything in particular you had in mind?

Reply:
Could you add blank after comma?

Reply:
I ran formatter on the files changed in this PR, so I'm good with the formatting